pkgs/racket-doc/scribblings/guide/begin.scrbl

```diff
@@ -64,6 +56,22 @@ positions, instead of forming an expression, the content of
 This splicing behavior is mainly useful for macros, as we discuss
 later in @secref["macros"].
 
+The whole @racket[_body] sequence is sometimes said to have an @deftech{implicit begin}.
+The restriction of an @tech{implicit begin} follows the restriction of the @racket[_body] sequence:
+the sequence should end with an expression.
+
```

Comment:
An implicit begin is restricted in the same way as a @racket[_body]; it must end with an expression.